gcc 11 rejects `class SLANG_EXPORT [[nodiscard]]` when SLANG_EXPORT expands
to an attribute: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=69585

diff --git a/slang-12.0/external/BS_thread_pool.hpp b/slang-12.0/external/BS_thread_pool.hpp
--- a/slang-12.0/external/BS_thread_pool.hpp
+++ b/slang-12.0/external/BS_thread_pool.hpp
@@ -968,7 +968,7 @@ inline bool set_os_process_priority(const os_process_priority priority)
 /**
  * @brief A class used to obtain information about the current thread and, if native extensions are enabled, get/set its priority, affinity, or name.
  */
-class SLANG_EXPORT [[nodiscard]] this_thread
+class SLANG_EXPORT this_thread
 {
     template <tp>
     friend class thread_pool;
